.circleci/config.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+jobs:
+    build:
+      docker:
+        - image: praqma/helm-aws #Change to helm-gcs
+      steps:
+        - checkout
+        - run: chmod +x .circleci/build.sh
```

Comment:
you can do the `chmod +x .circleci/build.sh` on your machine and push it. Then you can remove this line from the pipeline config